src/build.d

```diff
@@ -248,6 +281,7 @@ auto defineBackend()
 {
     // Pack the backend
     Dependency dependency = {
+        name: "backend",
```

Comment:
Please add a description.  And maybe the name could be more descriptive; `pack_backend`, `archive_backend`, `lib_backend`?

Reply:
The names are already pre-existing.  We could change them but to be consitent we'd probably want to change them an a bunch of other places like the Makefile/Documentation/etc.